File: inheritance.cpp
#include <iostream>
#include <string>
using namespace std;

// Parent Class
class Electronic {
public:
    string brand = "ASUS";
    void powerOn() {
        cout << "Device is booting up..." << endl;
    }
};

// Child Class inherits from Electronic
class Laptop : public Electronic {
public:
    string model = "Vivobook";
};

int main() {
    Laptop myLaptop;
    myLaptop.powerOn(); // Accessing parent function
    cout << myLaptop.brand << " " << myLaptop.model << endl;
    return 0;
}

File: inheritance.py
# Parent Class
class Electronic:
    brand = "ASUS"
    def power_on(self):
        print("Device is booting up...")

# Child Class inherits from Electronic
class Laptop(Electronic):
    model = "Vivobook"

my_laptop = Laptop()
my_laptop.power_on() # Accessing parent function
print(f"{my_laptop.brand} {my_laptop.model}")